test/test_diff_2.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
```

Comment:
Some safe options to set for a Bash script:
```sh
set -o errexit
set -o nounset
set -o pipefail
```
[Here](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html) a comprehensive list of the options you can use.

Reply:
I did not know about that, I should like put it on the beginning of each script ?